datadog/dogshell/__init__.py

```diff
@@ -71,6 +71,14 @@ def main():
         "-v", "--version", help="Dog API version", action="version", version="%(prog)s {0}".format(__version__)
     )
 
+    parser.add_argument(
+        "--site",
+        help="Datadog site to send data, us (datadoghq.com), eu (datadoghq.eu) or us3 (us3.datadoghq.com), default: us",
+        dest="site",
+        default=os.environ.get("DD_SITE", os.environ.get("DATADOG_HOST")),
+        choices=["datadoghq.com", "us", "datadoghq.eu", "eu", "us3.datadoghq.com", "us3"],
```

Comment:
Are you sure we want support `us` and `eu` ? That could be confusing with other datacenters.

Reply:
I thought we can be aligned with https://github.com/DataDog/datadogpy/blob/master/datadog/dogshell/wrap.py#L289
What do you think? Should we remove `us`, `eu`, `us3`?